cupy/cuda/runtime.pxd

```diff
@@ -309,6 +309,8 @@ cpdef int runtimeGetVersion() except? -1
 
 cpdef int getDevice() except? -1
 cpdef int deviceGetAttribute(int attrib, int device) except? -1
+cpdef int deviceGetByPCIBusId(str pci_bus_id) except? -1
+cpdef unicode deviceGetPCIBusId(int device)
```

Comment:
How about `str` instead of `unicode`?

Reply:
I used `unicode` because `str` means something different in Python 2 (`bytes`) and Python 3 (`unicode`). I made a choice with `unicode`, but maybe `bytes` would be better?

Reply:
Actually, thinking about it more, leaving it as `str` is probably a good idea as Python 2 is EOL in 3 months.

Reply:
I suppose that's right. CuPy has dropped the support of Python 2 in v7.

Reply:
Either way, CuPy uses cython in language level 3. `str` should be treated as `unicode`.